simcon_project/simcon_project/urls.py

```diff
@@ -19,11 +19,13 @@
 from django.conf.urls import include
 from django.contrib.auth import views
 
+app_name = 'simcon'
```

Comment:
do we want it to be simcon or simulatedConversations

Reply:
I'm okay with either way, but chose simcon since it was shorter